tests/automation/setup/recovery_using_seed.ts

```diff
@@ -1,12 +1,17 @@
 import { Page } from '@playwright/test';
+// eslint-disable-next-line import/no-cycle
 import {
   clickOnTestIdWithText,
   doesElementExist,
   typeIntoInput,
   waitForLoadingAnimationToFinish,
 } from '../utilities/utils';
 
-export async function recoverFromSeed(window: Page, recoveryPhrase: string) {
+export async function recoverFromSeed(
+  window: Page,
+  recoveryPhrase: string,
+  createDisplayName?: boolean,
+) {
```

Comment:
is this `createDisplayname` used anywhere currently? I feel like recoverFromSeed should fail, if the displayName isn't found, and not blindly offer the option to use "Alice" as one. Why was this needed?